src/features/views/components/ViewColumnDialog/SurveyResponseConfig.tsx

```diff
@@ -0,0 +1,226 @@
+import { useQuery } from 'react-query';
+import { useRouter } from 'next/router';
+import ZUIQuery from 'zui/ZUIQuery';
+import {
+  Autocomplete,
+  FormControl,
+  ListSubheader,
+  MenuItem,
+  TextField,
+  Typography,
+} from '@mui/material';
+import { ChangeEventHandler, useState } from 'react';
+import { FormattedMessage, FormattedMessage as Msg, useIntl } from 'react-intl';
+
+import getSurveysWithElements from 'features/smartSearch/fetching/getSurveysWithElements';
+import { COLUMN_TYPE, SelectedViewColumn } from '../types';
+import {
+  ELEMENT_TYPE,
+  RESPONSE_TYPE,
+  ZetkinSurveyElement,
+} from 'utils/types/zetkin';
+
+interface SurveyResponseConfigProps {
+  onOutputConfigured: (columns: SelectedViewColumn[]) => void;
+}
+
+export enum SURVEY_QUESTION_OPTIONS {
+  ALL_OPTIONS = 'AllSingle',
+  ALL_OPTIONS_SEPARATED = 'AllOptionsSeparated',
+  ONE_OPTION = 'oneOption',
+}
+
+const SurveyResponseConfig = ({
+  onOutputConfigured,
+}: SurveyResponseConfigProps) => {
+  const { orgId } = useRouter().query;
+  const surveysQuery = useQuery(
+    ['surveysWithElements', orgId],
+    getSurveysWithElements(orgId as string)
+  );
+  const intl = useIntl();
+  const columnsReset: SelectedViewColumn[] = [];
```

Comment:
This variable seems unnecessary. It's only used in one place, and the literal `[]` could be used there instead.